wrappers/python/indy_vdr/ledger.py

```diff
@@ -760,7 +761,9 @@ def prepare_txn_author_agreement_acceptance(
     if not accepted_time:
         # rough timestamp
         accepted_time = int(
-            datetime.combine(date.today(), datetime.min.time()).timestamp()
+            datetime.combine(
+                date.today(), datetime.min.time(), timezone.utc
```

Comment:
Could you change this to datetime.timezone.utc? This library doesn't import pytz.

Reply:
@andrewwhitehead Yes, changed, Thanks! Let me know if there are any other changes needed.